zorg/buildbot/builders/annotated/libc-linux.py

```diff
@@ -72,11 +68,10 @@ def main(argv):
         if lint_build:
             cmake_args.append('-DLLVM_LIBC_CLANG_TIDY=%s' % clang_tidy)
 
-        if runtimes_build:
-          projects = ['llvm', 'clang']
-          cmake_args.append('-DLLVM_ENABLE_RUNTIMES=libc')
-        else:
-          projects = ['llvm', 'libc']
+        # TODO: remove once old hdrgen is deleted.
+        # https://github.com/llvm/llvm-project/pull/117220
```

Comment:
https://github.com/llvm/llvm-project/pull/117220 has landed, so I can strip that out.

Reply:
d43d4d2d91ab